pkg/client-sdk/client_test.go

```diff
@@ -0,0 +1,378 @@
+package arksdk
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/ark-network/ark/pkg/client-sdk/client"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestVtxosToTxs(t *testing.T) {
+	tests := []struct {
+		name    string
+		fixture string
+		want    []Transaction
+		wantErr assert.ErrorAssertionFunc
+	}{
+		{
+			name:    "Alice Before Sending Async",
+			fixture: aliceBeforeSendingAsync,
+			want: []Transaction{
+				{
+					TxID:    "377fa2fbd27c82bdbc095478384c88b6c75432c0ef464189e49c965194446cdf",
+					Amount:  20000,
+					Type:    TxReceived,
+					Pending: false,
+					Claimed: false,
+				},
+			},
+			wantErr: assert.NoError,
+		},
+		{
+			name:    "Alice After Sending Async",
+			fixture: aliceAfterSendingAsync,
+			want: []Transaction{
+				{
+					TxID:    "377fa2fbd27c82bdbc095478384c88b6c75432c0ef464189e49c965194446cdf",
+					Amount:  20000,
+					Type:    TxSent,
+					Pending: false,
+					Claimed: false,
+				},
+				{
+					TxID:    "94fa598302f17f00c8881e742ec0ce2f8c8d16f3d54fe6ba0fb7d13a493d84ad",
+					Amount:  19000,
+					Type:    TxReceived,
+					Pending: true,
```

Comment:
let's add a TODO to remember that this will change once the ASP handles the change of an async payment as a normal vtxo:
```suggestion
					Pending: true, // TODO: expect false once the ASP handles the change properly
```

Reply:
Done